ci/circle

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python2.7
```

Comment:
I have a couple projects now with a CI/ directory (capital letters).
Unless you feel strongly, it would be nice to match.